templates/addons/env/rds-postgres.yml

```diff
@@ -73,7 +73,7 @@ Resources:
       AutoMinorVersionUpgrade: true
       EnablePerformanceInsights: true
       Engine: postgres
-      EngineVersion: '13.4'
+      EngineVersion: '13'
```

Comment:
Not for this pull request, but do we want to make this something that developer teams can specify themselves?

Reply:
[DBTP-336 Allow overriding of default Postgres version](https://uktrade.atlassian.net/browse/DBTP-336)